QUERIES.md

```diff
@@ -40,8 +43,14 @@ REVOKE ALL [PRIVILEGES] ON <db> FROM <user>
 -- revoke all of user's privileges (all DBs and/or cluster admin)
 REVOKE ALL [PRIVILEGES] FROM <user>
 
+-- combine db creation with privilege assignment (user must already exist)
+CREATE DATABASE <name> GRANT <privilege> TO <user>
```

Comment:
Thanks, I hadn't checked specific permission levels. I'll verify the behavior and fix the docs